samples/wfi32iot/README.md

```diff
@@ -16,7 +16,64 @@ This document describes how to connect the WFI32-IoT Development Board (featurin
 Download the [WFI32-IoT Development Board User Guide](https://www.microchip.com/content/dam/mchp/documents/WSG/ProductDocuments/UserGuides/EV36W50A-WFI32-IoT-Board-Users-Guide-DS50003262.pdf) for more details including the schematics for the board (but do not follow the setup procedure in the document)
 
 ## Adding Extra Sensors to the WFI32 IoT Board
-Even though each WFI32E01 Development Board has its own on-board light and temperature sensors, additional sensors can optionally be added relatively quickly using existing off-the-shelf hardware.  [Click here](https://github.com/MicrochipTech/AzureDemo_WFI32E01/blob/main/Clicks.md) for more details on how to add these sensors!
+Even though each WFI32E01 Development Board has its own on-board light and temperature sensors, additional sensors can optionally be added relatively quickly using existing off-the-shelf hardware. 
+
+The WFI32E01 Development Board, like many Microchip development boards, features a 16-pin (2 rows x 8 pins) expansion socket which conforms to the [mikroBUS™ specification](https://download.mikroe.com/documents/standards/mikrobus/mikroBUS-standard.pdf). The mikroBUS™ standard defines mainboard sockets used for interfacing microcontrollers or microprocessors (mainboards) with integrated circuits and peripheral modules (add-on boards).
+
+The standard specifies the physical layout of the mikroBUS™ pinout, the communication and power supply pins used, the positioning of the mikroBUS™ socket on the mainboard, and finally, the silkscreen marking conventions for both the sockets. The purpose of mikroBUS™ is to enable easy hardware expandability with a large number of standardized compact add-on boards, each one carrying a single sensor, transceiver, display, encoder, motor driver, connection port, or any other electronic module or integrated circuit. Created by [MikroElektronika](https://www.mikroe.com), mikroBUS™ is an open standard — anyone can implement mikroBUS™ in their hardware design.
+
+<img src=".//media/image8.png" style="width:4in;height:5in"/>
```

Comment:
There is a way to display an embedded image in markdown syntax. We are doing it with several boards in QUICKSTART_DEMO.md in this proejct.Do we really _need_ this html or do you need to control whis widht/height etc. 

We also should not need <br>. Space should be automatic between paragraphs.